### DISCUSSION TITLE
Support doesn't reply

### DISCUSSION THREAD

**Discussion by ghost** [coreui/coreui-vue#197] [General]
Hi,

as discussed before, I am not getting any response from your support. So when I post a question to https://coreui.io/support/, there is no reply email. I checked with our IT department, and they claim there were no emails filtered. Might be an issue on your end?

Thanks.

**Comment by xidedix** [comment]
Hello, 

We've found your email dated March 16th. It was slightly unreadable due to formatting issues, but that's on our side.  
You raised four questions. Ticket #2811. We'll get back to you.
